app/components/product/alias-form.hbs

```diff
@@ -0,0 +1,32 @@
+<ul>
+  {{#each @aliases as |alias|}}
+    <li>
+    <div class="flex">
+      <div class="text-sm my-2 text-gray-900">{{alias}}</div>
```

Comment:
In edit mode, I would display all aliases as editable input-fields.

Reply:
I'm going to rely on your insights. OK to go ahead with the read-only list as it is now :+1: